Adding a summary of scaling and external application tests to readme (#55)
quarkus-qe/quarkus-openshift-test-suite: README.md

### `scaling`

> A simple OpenShift test verifying that the OpenShift deployment with Quarkus application scales up and down.

Reviewer: ```suggestion
An OpenShift test verifying that the OpenShift deployment with Quarkus application scales up and down.
```

Reviewer 2: I'd even suggest just "Verifies that an OpenShift deployment with Quarkus application scales up and down."

Reviewer 2: And perhaps add that this could be expanded in the future to collect some performance metrics -- but that's hard to do in a meaningful way, because the OpenShift overhead is much bigger than Quarkus startup time.

### `external-applications/todo-demo-app`

This test produces a S2I source deployment config for OpenShift with todo-demo-app serving a simple todo checklist.
The test verifies that the application with a sample of libraries is buildable and deployable via supported means.

### `external-applications/quarkus-workshop-super-heroes`

This test produces a S2I source deployment config for OpenShift with Quarkus Super heroes workshop application.
The test verifies that the application is buildable and deployable. It also verifies that the REST and MicroProfile APIs
function properly on OpenShift, as well as the database integrations.

> 

Reviewer: State explicitly that source code of todo-demo-app and quarkus-workshop-super-heroes lives outside quarkus-openshift-test-suite.

Reviewer 2: +1.

Also, "_an_ S2I source ..."

### `scaling`

An OpenShift test verifying that an OpenShift deployment with a Quarkus application scales up and down.

This test could be extended with some metric gathering.

### `external-applications/todo-demo-app`

This test produces an S2I source deployment config for OpenShift with [todo-demo-app](https://github.com/quarkusio/todo-demo-app) 
serving a simple todo checklist. The code for this application lives outside of the test suite's codebase.

The test verifies that the application with a sample of libraries is buildable and deployable via supported means.

### `external-applications/quarkus-workshop-super-heroes`

This test produces an S2I source deployment config for OpenShift with 
[Quarkus Super heroes workshop](https://github.com/quarkusio/quarkus-workshops) application.
The code for this application lives outside of the test suite's codebase.

> ''

Reviewer 2: ? :-)

Reviewer: up, I missed that, but already merged :(

Reviewer: I will prepare micro pr for it ;)

Reviewer: https://github.com/quarkus-qe/quarkus-openshift-test-suite/pull/56
